hamlpy/test/test_attributes.py

```diff
@@ -106,9 +106,16 @@ def test_read_attribute_dict(self):
         assert dict(self._parse("{class: 'test\u1234'}")) == {'class': 'test\u1234'}
 
         # html style dicts
+        assert dict(self._parse("()")) == {}
+        assert dict(self._parse("(   )")) == {}
+
+        assert dict(self._parse(
+            "(disabled :class='test' data-number = 123\n 'foo'=\"bar\")"
+        )) == {'disabled': None, 'class': 'test', 'data-number': '123', 'foo': 'bar'}
+
         assert dict(self._parse(
-            "(:class='test' :data-number = 123\n foo=\"bar\")"
-        )) == {'class': 'test', 'data-number': '123', 'foo': 'bar'}
+            "(:class='test' data-number = 123\n 'foo'=\"bar\"  \t   disabled)"
```

Comment:
WTF that is the ugliest syntax I've ever seen, what an abomination! Wonder what the reasoning around that is.

My bias for anything like this is to be strict instead of ignoring things that look goofy.